app/bundles/CampaignBundle/Resources/views/Event/_preview.html.twig

```diff
@@ -39,6 +49,12 @@ data-event-id="{{ event['id'] }}">
                 <span class="fs-18 fw-l">{{ ('mautic.campaign.'~event['type'])|trans }}</span>
                 <span class="campaign-event-name ellipsis">{% if ('dev' == constant('MAUTIC_ENV')) %}<small>{{ event['id'] }}</small> {% endif %}{{ event['name'] }}</span>
             </div>
+            <div>
+                <span class="label label-success" data-toggle="tooltip" title="{{ 'mautic.report.campaign.completed.actions'|trans }}"><span class="ri-user-6-fill"></span> {{ event.logCountProcessed }}</span>
+                {% if (event.logCountForPending) %}
+                <span class="label label-default" data-toggle="tooltip" title="{{ 'mautic.report.campaign.pending.actions'|trans }}"><span class="ri-hourglass-line"></span> {{ event.logCountForPending }}</span>
+                {% endif %}
+            </div>
```

Comment:
```suggestion
            <div class="campaign-event-stat">
                <span data-toggle="tooltip" title="{{ 'mautic.report.campaign.completed.actions'|trans }}"><span class="ri-check-line"></span> {{ event.logCountProcessed }}</span>
                {% if (event.logCountForPending) %}
                <span data-toggle="tooltip" title="{{ 'mautic.report.campaign.pending.actions'|trans }}">| <span class="ri-hourglass-line"></span> {{ event.logCountForPending }}</span>
                {% endif %}
            </div>
```

![image](https://github.com/user-attachments/assets/c913e4bf-9417-434c-95cb-96d4bc3856d2)

This change could make the presentation cleaner to users

Reply:
It looks nice, but what about the conditions and decisions? Should we change them to keep the statistics consistent? Do you have any idea where to place it?

Reply:
Oh, the idea was to apply this to all events, did I miss something in the code?

Reply:
IMO only the ideal case scenario with action looks good. Even events with longer names will overlap the counters. Have a look
![image](https://github.com/user-attachments/assets/714699ff-d8f9-42dc-8e42-aeb243d64ba8)

Reply:
Right, let's go with the last working solution 

![image](https://github.com/user-attachments/assets/22984535-cd5a-4b19-88ea-66d6f056316d)

Reply:
@patrykgruszka tell me when it's ready, I'll do the user testing

Reply:
@andersonjeccel thanks! It's ready to test.